src/Service/MetabaseService.php

```
@@ -0,0 +1,86 @@
+
+namespace App\Service;
+
+use App\DTO\RedmineTicket;
+use App\DTO\RedmineTicketList;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Component\DependencyInjection\Attribute\Autowire;
+use Symfony\Component\Serializer\SerializerInterface;
+
+readonly class MetabaseService
+{
+    private SerializerInterface $serializer;
```

PHPStan (2.0.3)

Property App\Service\MetabaseService::$serializer is never read, only written.
See: https://phpstan.org/developing-extensions/always-read-written-properties

(property.onlyWritten)